Engine/DataFeeds/Transport/RestSubscriptionStreamReader.cs

```diff
@@ -75,6 +79,7 @@ public string ReadLine()
                 var response = _client.Execute(_request);
                 if (response != null)
                 {
+                    _delivered = !_isLiveMode;
```

Comment:
Should this care about live mode? `EndOfStream` properly checks both flags, not sure we need to differentiate between live/non-live on this line.